composer.json

```diff
@@ -26,7 +26,7 @@
         "doctrine/persistence": "^1.3",
         "sonata-project/datagrid-bundle": "^2.5",
         "sonata-project/doctrine-extensions": "^1.5",
-        "sonata-project/form-extensions": "^1.2",
+        "sonata-project/form-extensions": "^0.1 || ^1.4",
```

Comment:
@jordisala1991 this time I have to leave the left part, otherwise Composer cannot resolve the constraints.

Reply:
Probably because other sonata bundles are not released and you cant install 1.x

Reply:
You should not allow install 0.x in Sonata 4. After release all Sonata 3 bundles this will be work without it or we can delete 0.x before release sonata 4.